script/github-release

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
```

Comment:
> For instance, something that hub doesn't let me do right now is upload the compiled asset even if it exists.

This is valid point. It would be nice to support this in the future.